packages/replay-internal/test/integration/errorSampleRate.test.ts

```diff
@@ -364,24 +365,6 @@ describe('Integration | errorSampleRate', () => {
           },
         ]),
       });
-
-      vi.advanceTimersByTime(DEFAULT_FLUSH_MIN_DELAY);
```

Comment:
just removed this because it actually irrelevant to this specific test.

Reply:
Does this behavior get covered in another test?